build.targets.ps1

```diff
@@ -195,4 +195,5 @@ function RunTestWithCoverage($fullTestDllPaths) {
         Write-Host -ForegroundColor DarkBlue "Uploading coverage report to Coveralls.io"
         Exec { . $converallsNetExe --opencover $outputXml }
     }
+}
```

Comment:
Not an intentional change. Must have been a merge issue. This change can be reverted.